src/actions/index.ts

```diff
@@ -55,6 +55,29 @@ export namespace Actions {
     })
   }
 
+  export namespace WorkClaimForm {
+    export const SUBMIT = 'SUBMIT_WORK_CLAIM_FORM'
+    export const SUBMIT_SUCCESS = 'SUBMIT_WORK_CLAIM_FORM_SUCCESS'
+    export const SUBMIT_ERROR = 'SUBMIT WORK_CLAIM_FORM_ERROR'
+    export const SUBMIT_CLEAR_ERROR = 'SUBMIT WORK_CLAIM_FORM_CLEAR_ERROR'
```

Comment:
```suggestion
    export const SUBMIT_CLEAR_ERROR = 'SUBMIT_WORK_CLAIM_FORM_CLEAR_ERROR'
```

Reply:
the action would not create a work or claim? Why not we can change to for example:

```
 export namespace CreateClaim {
    export const CREATE_CLAIM = 'CREATE_CLAIM'
    export const CREATE_CLAIM_SUCCESS = 'CREATE_CLAIM_SUCCESS'
    export const CREATE_CLAIM_ERROR = 'CREATE_CLAIM_ERROR'
    export const CREATE_CLAIM_CLEAR_ERROR = 'CREATE_CLAIM_CLEAR_ERROR'
```

Reply:
That does seem to follow the pattern of this app better.

Reply:
Ok I went ahead and updated it to CreateClaim.